### ISSUE TITLE
[Darc CLI] Get subscriptions

### ISSUE THREAD

**Issue by mmitche** [dotnet/arcade#868]
Implement get-subscriptions command.  Should have the following:
- List all subscriptions by default
- List specific subscriptions by the following constraints (may be combined)
  - Target Repo
  - Source channel
  - Target branch

**Comment by mmitche** [comment]
@jcagme fyi

**Comment by mmitche** [comment]
Maybe also way to get information about when sub last executed, etc

**Comment by jcagme** [comment]
Yeah, we could use the history API. Also probably looking for subscriptions tied to builds which failed also in the history API

**Comment by mmitche** [comment]
I'm going to implement a separate get-subscription command which takes the ID of a subscription and presents the history.  I'll highlight what the history command is in the output.
